{{cookiecutter.__project_name}}/.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - develop
+      - main
```

Comment:
Just a nitpick suggestion to match the order of the `pull_request` branches list.
```suggestion
      - main
      - develop
```